docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.8'
+
+services:
+  app:
+    build: ./go-server
+    ports:
+      - "8080:8080"
+    environment:
+      - DB_URI=mongodb://db:27017
```

Comment:
what is the reason to have in .env and here?

Reply:
It's hard to explain, it was swearing at something, that's why I added it.

Reply:
Please either explain it or remove it. Because it's odd